__tests__/app.test.js

```diff
@@ -36,3 +36,24 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET: 20, responds with corresponding article object when given an article_id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article[0].article_id).toBe(1);
+        expect(article[0].title).toBe("Living in the shadow of a great man");
+        expect(article[0].topic).toBe("mitch");
+        expect(article[0].author).toBe("butter_bridge");
+        expect(article[0].body).toBe("I find this existence challenging");
+        expect(article[0].created_at).toBe("2020-07-09T20:11:00.000Z");
+        expect(article[0].votes).toBe(100);
+        expect(article[0].article_img_url).toBe(
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        );
+      });
+  });
+});
```

Comment:
This is a lovely happy path test 😁 But there are a couple of error handling tests missing:

- What if the article id is 9000?
- What if the article id is not a number?